AlphaWallet/Tokens/Coordinators/SingleChainTokenCoordinator.swift

```diff
@@ -327,19 +328,22 @@ class SingleChainTokenCoordinator: Coordinator {
         )
         addCoordinator(tokensCardCoordinator)
         tokensCardCoordinator.delegate = self
-        tokensCardCoordinator.start()
+
         switch (type, session.account.type) {
         case (.send, .real), (.request, _):
```

Comment:
I think we need the `tokensCardCoordinator.start()` line needs to be moved into the switch so it gets called for the first 2 cases, but not the 3rd/last?